setup.py

```diff
@@ -56,6 +56,12 @@
           'schema-salad >= 2.6.20170927145003, < 3',
           'typing >= 3.5.3',
           'six >= 1.8.0',
+          'prov == 1.5.1',
+          'pathlib == 1.0.1',
+          'graphviz == 0.8.2',
+          'pygraphviz == 1.3.1',
+          'Matplotlib == 2.1.2',
+          'pydot == 1.2.4'
```

Comment:
Likewise pydot doesn't appear to be used either.

Reply:
Removed